Libraries/Text/Text/RCTTextView.m

```diff
@@ -423,22 +493,19 @@ - (BOOL)becomeFirstResponder
 
 - (BOOL)resignFirstResponder
 {
-  if (![super resignFirstResponder]) {
+  //  Don't relinquish first responder while selecting text.
+  if (_selectable && NSRunLoop.currentRunLoop.currentMode == NSEventTrackingRunLoopMode) {
     return NO;
   }
-
-  // If we've lost focus, notify listeners
-  [_eventDispatcher sendEvent:[RCTFocusChangeEvent blurEventWithReactTag:self.reactTag]];
-
-  return YES;
+  
+  return [super resignFirstResponder];
 }
 
-#endif // ]TODO(macOS GH#774)
-
 - (BOOL)canBecomeFirstResponder
 {
-  return _selectable;
+  return self.focusable;
```

Comment:
Use `focusable` prop to set first responder. if `_selectable` is true, `focusable` will be true